dev/com.ibm.ws.jaxrs.2.0.common/src/com/ibm/ws/jaxrs20/security/LibertySimpleAuthorizingInterceptor.java

```diff
@@ -29,17 +31,24 @@
 import com.ibm.websphere.ras.Tr;
 import com.ibm.websphere.ras.TraceComponent;
 
+@Priority(Priorities.AUTHORIZATION)
 public class LibertySimpleAuthorizingInterceptor extends
                 AbstractAuthorizingInInterceptor {
     private static final TraceComponent tc = Tr
                     .register(LibertySimpleAuthorizingInterceptor.class);
 
     @Override
     public void handleMessage(Message message) throws Fault {
-        SecurityContext sc = message.get(SecurityContext.class);
-        if (sc != null) {
+        SecurityContext cxfSecurityContext = message.get(SecurityContext.class);
+        javax.ws.rs.core.SecurityContext jaxrsSecurityContext = message.get(javax.ws.rs.core.SecurityContext.class);
+        if (jaxrsSecurityContext != null && jaxrsSecurityContext instanceof javax.ws.rs.core.SecurityContext) {
```

Comment:
```suggestion
        if (jaxrsSecurityContext != null) {
```

The instanceof check is unnecessary.